locales/base/translation.json

```diff
@@ -1076,8 +1077,8 @@
   "feedItemCeloRewardReceivedTitle": "Earned CELO",
   "feedItemRewardReceivedTitle": "Supercharge",
   "feedItemRewardReceivedInfo": "Weekly Rewards",
-  "feedItemInviteRewardReceivedTitle": "Celo Network",
-  "feedItemInviteRewardReceivedInfo": "Reward Received",
+  "feedItemInviteRewardReceivedTitle": "{{appName}} Invite Reward",
+  "feedItemInviteRewardReceivedInfo": "Payment Received",
```

Comment:
now i'm not sure about these guys but i think they'll do for now, we can push via OTA if it's not right. noone is online to confirm >.<

Reply:
Per the article: https://docs.google.com/document/d/1I8xkD9ztLUnkwqvKbipsqRfMEErQBvHkQxcapWoIH9s/edit#, looks like this needs to be `Valora - Rewards Received`

Reply:
oh good one, i missed this! i have updated :)